report_ai/index.html

```diff
@@ -44,7 +44,10 @@
       </div>
   </div>
   <div class="topbar">
-    <div style="width: 59%; text-align:right; padding-right:2%">Overall Page Rating</div>
+    <div class="container">
+      <div style="width: 59%; text-align:right; padding-right:2%">Overall Page Rating</div>
+      <div style="text-align:right" id = "numReviews"></div>

```

Comment:
Suggesting a change in the placement of the text. Put the number of users who reviewed below the `average review` 

![Screenshot 2023-10-23 at 11 25 37 AM](https://github.com/gobbledy-gook/report.ai/assets/96362727/76338478-c7eb-42d3-b13d-d3809738bede)

Something of this sort would be better.